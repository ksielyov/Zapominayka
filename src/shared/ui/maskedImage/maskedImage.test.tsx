import React from 'react';
import renderer from 'react-test-renderer';
import {MaskedImage} from '@ui';

test('maskedImage should render correctly', () => {
  const tree = renderer
    .create(
      <MaskedImage
        imageSource={
          'https://moreulybok.ru/wp-content/uploads/2021/05/aleksandr-pushkin7.jpg'
        }
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
