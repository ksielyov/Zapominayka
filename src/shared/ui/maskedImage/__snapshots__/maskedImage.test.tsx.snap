// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`maskedImage should render correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight={88.27941176470588}
  bbWidth={88.27941176470588}
  fill="none"
  focusable={false}
  height={88.27941176470588}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      Object {
        "flex": 0,
        "height": 88,
        "width": 88,
      },
    ]
  }
  vbHeight={52}
  vbWidth={52}
  width={88.27941176470588}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      Array [
        "fill",
      ]
    }
  >
    <RNSVGDefs>
      <RNSVGPattern
        height="52"
        matrix={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        name="imageSource"
        patternContentUnits={1}
        patternTransform={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        patternUnits={1}
        width="52"
        x="0%"
        y="0%"
      >
        <RNSVGImage
          align="xMidYMid"
          height="52"
          meetOrSlice={0}
          src={
            Object {
              "uri": "https://moreulybok.ru/wp-content/uploads/2021/05/aleksandr-pushkin7.jpg",
            }
          }
          width="52"
          x="0"
          y="0"
        />
      </RNSVGPattern>
    </RNSVGDefs>
    <RNSVGPath
      d="M26.088 52a112 112 0 01-4.67-.094c-6.788-.283-15.339-1.231-17.54-4.139C1.635 44.806.246 34.777.03 26.4-.136 19.616.26 7.636 4.935 3.634 9.173.004 24.732-.472 34.432.338c7.74.655 12.529 2.03 14.238 4.087 1.177 1.413 2.039 4.261 2.618 8.675.517 4.312.753 8.653.706 12.995-.026 4.758-.334 9.381-.87 13.018-.658 4.45-1.604 7.31-2.894 8.748C45.3 51.131 34.436 52 26.088 52z"
      fill={
        Array [
          1,
          "imageSource",
        ]
      }
      propList={
        Array [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
